helper/common/common.go

```diff
@@ -174,3 +185,48 @@ func PadLeftOrTrim(bb []byte, size int) []byte {
 
 	return tmp
 }
+
+// FetchWhitelistFromConfig fetches whitelist object from the config
+// if doesn't exist returns empty map
+func FetchWhitelistFromConfig(genesisConfig *chain.Chain) map[string]interface{} {
```

Comment:
How about defining these 2 helper functions in the different file but under the same package?

Reply:
in which file would you put it?

Reply:
I meant to create file in the same directory

Reply:
Fixed ba491dd6391edb51e4dc30c0c0dfee8b95d60f93